src/app/service/todo-http.service.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ToDoItem } from 'src/model/ToDoItem';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TodoHttpService {
+  constructor(private httpClient: HttpClient) {}
+  getAll() {
+    return this.httpClient.get<ToDoItem[]>(`https://localhost:44309/ToDoItems`);
```

Comment:
can extract a  baseURL=`https://localhost:44309`.Will be much easier when make changes